assets/js/app.js

```diff
@@ -1,74 +1,12 @@
 'use strict';
 
-const moment = require('moment');
-
-const HUMAN_FORMAT = 'D/M/YYYY';
-const SERVER_FORMAT = 'YYYY-MM-DD HH:mm:ss:SSS';
-
-const submit = document.getElementById('submit');
-const blockForm = () => submit.disabled = true;
-const unblockForm = () => submit.disabled = false;
-const formBlocked = () => submit.disabled;
-
-const stateChanged = (dateDescription, xhttp) => () => {
-  if (xhttp.readyState === 4) {
-    if (xhttp.status === 200) {
-      const data = JSON.parse(xhttp.responseText);
-      // const layout = {
-      //   title: `LEV usage ${dateDescription}`,
-      //   barmode: 'stack'
-      // };
-
-      // here!
-      console.log(data);
-    } else {
-      console.log('HTTP ERROR:', xhttp.status, xhttp.statusText);
-    }
-    unblockForm();
-  } else {
-    console.log(`Update (${xhttp.readyState}):`, xhttp.status, xhttp.statusText);
-  }
+const React = require('react');
+const ReactDOM = require('react-dom');
+const DashboardContent = require('lev-react-components').DashboardContent;
+
+document.body.onload = () => {
+  ReactDOM.hydrate(
+    React.createElement(DashboardContent, window.dashboardProps, null),
+    document.getElementById('content')
```

Comment:
I think this should hydrate `LevReport` i.e. the whole App.
It should probably go into a `#root` element. That seems to be the standard.

Reply:
`window.dashboardProps` could become something more standard, like `window.appProps` or `window.props`. I wonder if there is a React standard for that.

Reply:
If I understand this right I think we should be able to avoid using the window object.

I think this can all be built into `lev-react-renderer`...

Reply:
I think for now we will keep this, until the `lev-react-renderer` has a more standard way of handling this.